src/sql/azdata.proposed.d.ts

```diff
@@ -725,6 +725,10 @@ declare module 'azdata' {
 			 * Extension can store additional information that the provider needs to uniquely identify a table.
 			 */
 			[key: string]: any;
+			/**
+			 * Table Node subtype (Temporal/GraphNode/GraphEdge)
+			 */
+			tableSubType: string;
```

Comment:
This seems like it should be an optional property - most tables aren't going to have a sub type

Reply:
Currently all tables are temporal or empty type and we default the icon for temporal table